### PR TITLE
Thick Clothing blocks borers and other tweaks

### PR DESCRIPTION
**Pull request by Super3222**
Thanks goes out to "David" from our discord page who suggested that borers shouldn't be able to breach bio suits.

### Intent of your Pull Request

- Borers cannot infect people wearing thick clothing. Via bio suits, the bee suit, spacesuits, etc
- Borers cannot speak while docile (when sugar is in their hosts blood).
- Borers can no longer use *collapse or *surrender
- Riot Armor now has a THICKMATERIAL flag, which will block borers.

#### Changelog

:cl:
fix: Borers can now longer use *collapse or *surrender.
tweak: Borers cannot speak while docile (when sugar is in their hosts blood)
tweak: Borers can no longer infect people wearing thick clothing on their head AND suit storage. (riot suits, bio suits, bee suits, hardsuits, spacesuits, etc)
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
I am glad we can still use **flip

Anyway, while you're at it, could you make protected headsets also borer proof?
Borers enter through the ears, so it would make sense if the headsets with protection, like sec headsets or earmufs would be protected against it. Would also prevent them from completely dominating security.

**Comment by ShadowDeath6** [comment]
no, because some dumbass bowman headset that allows you to hear at all should not block borers, and making sec entirely immune to them just ruins gameplay

**Comment by Time-Green** [comment]
They block out flashbangs but you can still hear around you.
It's safe to say there's protection in it and it covers your ears completely.
Also, I don't want the whole security team to be infected by fucking borers and walking around and force borering people with tasers.

**Comment by Super3222** [comment]
Maybe riot suits could have `THICKMATERIAL`? I don't see much harm in it. It'd also work perfectly by helping security contain borers.

File: code/modules/mob/living/simple_animal/borer/borer.dm [hunk 1]
```diff
@@ -127,6 +128,10 @@ var/total_borer_hosts_needed = 10
 					victim.say("*[pick(list("blink","blink_r","choke","aflap","drool","twitch","twitch_s","gasp"))]")
 
 /mob/living/simple_animal/borer/say(message)
+	if(docile)
+		src << "<span class='warning'>Your state is weakened by the nature of sugar within your hosts blood. You are too docile to even speak.</span>"
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer.dm]: Host's

File: code/modules/mob/living/simple_animal/borer/borer.dm [hunk 2]
```diff
@@ -138,6 +143,16 @@ var/total_borer_hosts_needed = 10
 		src << "<span class='warning'>You cannot speak without a host!</span>"
 		return
 	if(dd_hasprefix(message, "*"))
+		var/pass = TRUE
+		for(var/M in banned_borer_emotes)
+			if(M == message)
+				src << "<span class='warning'>You don't have the strength to do this...</span>"
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer.dm:148]: Borers never have the strength to do it, so display something like "we are unable to say this. "

Comment (Super3222) [reply code/modules/mob/living/simple_animal/borer/borer.dm:148]: "We"

Comment (ShadowDeath6) [reply code/modules/mob/living/simple_animal/borer/borer.dm:148]: Most ayy lamos use plural when referring to oneself

File: code/modules/mob/living/simple_animal/borer/borer.dm (hunk 1 above)
Comment (ShadowDeath6) [reply code/modules/mob/living/simple_animal/borer/borer.dm]: Also please remove this entire check as it's a massive balance change, muting borers means they can't get help or even tell their host to fuck off if they accidentally take sugar

Comment (Super3222) [reply code/modules/mob/living/simple_animal/borer/borer.dm]: Makes sense. I'll handle it in the next commit.

File: code/modules/mob/living/simple_animal/borer/borer.dm (hunk 2 above)
Comment (Super3222) [reply code/modules/mob/living/simple_animal/borer/borer.dm:148]: Eh, I want to direct the message to the player. I feel like using "we" would be for something like changelings.

**Comment by Super3222** [comment]
@Cruix This is ready for review (or merge - however you take it).

**Pull request merged**